db/sql/blocks/aggregateBlocksReward.sql

```diff
@@ -0,0 +1,41 @@
+/*
+  DESCRIPTION: ?
+
+  PARAMETERS: ?
+*/
+
+WITH delegate AS
+  (SELECT 1
+   FROM ${schema~}.mem_accounts m
+   WHERE m."isDelegate" = 1
+     AND m."publicKey" = decode($1, 'hex')
+   LIMIT 1),
+     rewards AS
+  (SELECT count(1) AS count,
+          sum(reward) AS rewards
+   FROM ${schema~}.blocks
+   WHERE "generatorPublicKey" = decode($1, 'hex')
+     AND ($2 IS NULL
+          OR TIMESTAMP >= $2)
```

Comment:
@vitaly-t `timestamp` is a column in `blocks` table, globally in that query.

Reply:
Good catch! Looks like another auto-format change by the SQL editor set me up. Will need to fix it.

Reply:
@4fryn On a second look, the discrepancy appears to be purely cosmetic, it doesn't affect the functionality, as `TIMESTAMP` isn't mistreated as a reserved word, it works the same as `timestamp` column ;)